src/content/structured/get-started/gatsby.mdx

```diff
@@ -7,20 +7,11 @@ date: "2024-04-24"
 
 title: "Gatsby"
 
-subTitle: "How to use the components in a Gatsby-based application."
+subTitle: "Lock the React components and corresponding web components package to the same version."
```

Comment:
this doesn't feel appropriate for the context of the page - it's not just about locking the components